src/docs/cookbook/networking/authenticated-requests.md

```diff
@@ -20,22 +22,22 @@ convenient way to add headers to your requests.
 Alternatively, use the [`HttpHeaders`][]
 class from the `dart:io` library.
 
-<!-- skip -->
+<?code-excerpt "lib/main.dart (get)"?>
 ```dart
-Future<http.Response> fetchAlbum() {
-  return http.get(
-    Uri.parse('https://jsonplaceholder.typicode.com/albums/1'),
-    // Send authorization headers to the backend.
-    headers: {HttpHeaders.authorizationHeader: "Basic your_api_token_here"},
-  );
-}
```

Comment:
I simplified a bit the example by removing the function declaration surrounding it